### ISSUE TITLE
Add also apps that kill other apps

### ISSUE THREAD

**Issue by thubalek** [urbandroid-team/dont-kill-my-app#49]
Would it be possible to include apps like Avast Cleanup (https://play.google.com/store/apps/details?id=com.avast.android.cleaner), that kill other apps? 

Some users may adjust their phone but app will be still killed by such a apps.

**Comment by petrnalevka** [comment]
Does anyone know if those apps really work? From the Android API docs it does not seem there is a direct way to kill another app without system permissions or on rooted phones. Are those apps just placebo or they really work somehow?

**Comment by thubalek** [comment]
Yes, they can really harm your app. They misuse _Accessibility permission_, open app settings and push button _Force close_ on behalf of the user. This is hidden by overlay drawn over the settings app so user don't see it.

This is approach invented by Greenify (https://play.google.com/store/apps/details?id=com.oasisfeng.greenify) many years ago and all those pseudosavers implement it in similar way.

**Comment by petrnalevka** [comment]
Yes, I have tested Avast cleaner and indeed it is force stopping apps which breaks their alarms etc..

So I guess we need to create a list of those apps and advice users to not use them or at least don't give them accessibility permission..

**Comment by hb0** [comment]
I notices a few days ago that a popular "App Lock" App I tried contains a "Batterysafer" Option which killed my Background Process (Foreground Service):

App name: "AppLock"
Play Store Link: https://play.google.com/store/apps/details?id=com.hyperspeed.rocket.applock.free
Feature: "Batterysafer" Option

But I think the list of such apps is very long.
